src/Shared/Server/infra/test/resources/index.js

```diff
@@ -0,0 +1 @@
+alert("hello world!")
```

Comment:
this is enough to check it?

Reply:
may be a description of what works rather than hello world would be better.

Reply:
This code doesn't even need to work, it's just there to check that the server serves the correct files. It could perfectly be a text file with random text